packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -324,6 +326,29 @@ export default class NewVm extends BaseComponent {
     })
   }
 
+  _selfCreate = () => {
+    const {
+      CPUs,
+      VDIs,
+      existingDisks,
+      memoryDynamicMax,
+      template,
+    } = this.state.state
+    const disksSize = sumBy(VDIs, 'size') + sumBy(existingDisks, 'size')
+    const templateDisksSize = sumBy(template.template_info.disks, 'size')
+    const templateMemoryDynamicMax = template.memory.dynamic[1]
+    const templateVcpusMax = template.CPUs.max
+
+    return CPUs > MULTIPLICAND * templateVcpusMax ||
+      memoryDynamicMax > MULTIPLICAND * templateMemoryDynamicMax ||
+      disksSize > MULTIPLICAND * templateDisksSize
```

Comment:
What if the user is creating multiple VMs at once?

Reply:
The VMs will have the same configuration, so I think this check is still valid.

Reply:
Right, we're talking "per VM".